src/BA_ABA_matrices.jl

```diff
@@ -94,6 +93,17 @@ function BA_Matrix(ybus::Ybus)
     return BA_Matrix(data, axes, lookup, subnetwork_axes, ybus.network_reduction_data)
 end
 
+function _get_series_susceptance(series_chain::Vector{Any})
+    series_susceptances = [_get_series_susceptance(segment) for segment in series_chain]
+    total_susceptance = 1 / (sum((1.0 ./ series_susceptances)))
+    return total_susceptance
+end
+function _get_series_susceptance(segment::PSY.Branch)
```

Comment:
We need to tighten this type because it can dispatch on DC Branches. Use ACTransmission